mlflow/tensorflow.py

```diff
@@ -389,6 +405,8 @@ def load_model(model_uri, tf_sess=None):
     """
     import tensorflow
 
+    _raise_deprecation_warning()
```

Comment:
I didn't add `_raise_deprecation_warning` in `autolog` because if we did, the warning would be raised when `mlflow.autolog` is called.